src/main/java/nextstep/lotto/dto/LottoTicket.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.lotto.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTicket {
+
+    private List<LottoNumber> ticket;
+
+    private LottoTicket(List<LottoNumber> numbers) {
+        this.ticket = new ArrayList<>();
+        for(LottoNumber number : numbers){
+            this.ticket.add(number);
+        }
```

Comment:
새 `ArrayList`를 만들고 add하지 않고 바로 할당해주어도 괜찮지 않을까요?

```suggestion
        this.ticket = numbers;
```

Reply:
아... 지금 보니... 굳이 ArrayList를 할당하지 않아도 되었네요 ㅎㅎ;;;;
이부분 수정하여 커밋하였습니다.